### PR TITLE
Make status editable even after marked as SOLVED

### PR DESCRIPTION
**Pull request by asdfryan**
`SOLVED` is only listed as an option for solved puzzles. (i.e. it is only possible to "SOLVE" a puzzle by adding an answer)

Screen Shot 2021-01-14 at 12 26 33 PM
Screen Shot 2021-01-14 at 12 26 43 PM

### CODE REVIEW

File: hunts/src/StatusCell.js [hunk 1]
```diff
@@ -7,8 +7,12 @@ import { updatePuzzle } from "./puzzlesSlice";
 export default function StatusCell({ row, value }) {
   const { id: huntId } = useSelector((state) => state.hunt);
   const dispatch = useDispatch();
-  if (["SOLVING", "STUCK", "EXTRACTION"].includes(value)) {
-    return (
+  var statuses_to_display = ["SOLVING", "STUCK", "EXTRACTION"];
```
Comment (erwa) [inline hunts/src/StatusCell.js:10]: It seems the convention is to use `const` for arrays, even if the _contents_ of them are modified, as long as the array variable itself is not reassigned.

https://stackoverflow.com/questions/46553062/what-is-the-preferred-declaration-convention-for-objects-or-arrays-const-or-let

File: hunts/src/StatusCell.js [hunk 2]
```diff
@@ -7,8 +7,12 @@ import { updatePuzzle } from "./puzzlesSlice";
...
+  if (value == "SOLVED") {
```
Comment (erwa) [inline hunts/src/StatusCell.js:11]: Is `value` always a string here? If so, we can use `===`

**Review by erwa — APPROVED** [review]
Nice! Just some minor nits.

File: hunts/src/StatusCell.js (hunk 1 above)
Comment (asdfryan) [reply hunts/src/StatusCell.js]: Thanks -- didn't know that. :)

**Review by rgossiaux — COMMENTED** [review]
On my phone but is there a way to mark it back as solved once if you un-mark it? Maybe we should add SOLVED to the options if row.original.answers or whatever it's called has at least one answer?

**Comment by asdfryan** [comment]
Right now you can only "solve" a puzzle by submitting an answer. Your
suggestion is a good idea, but not sure I have time to implement it before
7 :-/.

I'll create an issue.

**Pull request merged**